scripts/python_installer

```diff
@@ -105,10 +107,15 @@ function install_python() {
     echo_success_message " - Python $PYTHON_VERSION is installed at $PYENV_BIN"
 
     if python_is_not_in_path; then
-        print_path_export_instructions
+        if [ ${SUPPRESS_PATH_EXPORT_MESSAGE} = false ] ; then
+            print_path_export_instructions
+        fi
     else
+        PYTHON_ALREADY_IN_PATH=true
         echo ""
-        echo_with_indentation " - $PYENV_BIN is already in PATH no further action need be taken."
+        if [ ${SUPPRESS_PATH_EXPORT_MESSAGE} = false ] ; then
```

Comment:
Curious to know why you made this change.

Reply:
I'm essentially trying to not have the python installer say anything regarding PATH unless it's been run. Like above, this is just a check to see if the script should suppress the output of messages regarding PATH.